tsconfig.json

```
@@ -14,7 +14,8 @@
     "resolveJsonModule": true,
     "jsx": "react",
     "lib": ["es2019", "dom"],
-    "strictPropertyInitialization": true
+    "strictPropertyInitialization": true,
+    "noErrorTruncation": true
```

We don't want to commit this permanently
```suggestion
    "strictPropertyInitialization": true
```